Bigtable/src/Instance.php

```diff
@@ -0,0 +1,380 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient as InstanceAdminClient;
+use Google\Cloud\Bigtable\Admin\V2\Instance_Type;
+use Google\Cloud\Bigtable\Admin\V2\Instance_State;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\Cloud\Bigtable\Connection\ConnectionInterface;
+use Google\Cloud\Core\ArrayTrait;
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\LongRunning\LongRunningConnectionInterface;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+
+/**
+ * Represents a Cloud Bigtable instance
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Bigtable\BigtableClient;
+ *
+ * $bigtable = new BigtableClient();
+ *
+ * $instance = $bigtable->instance('my-instance');
+ * ```
+ *
+ * @method resumeOperation() {
```

Comment:
`resumeOperation()` doesn't exist in this PR.  It's an artifact of Cloud Spanner, not Cloud Bigtable

Reply:
I defer to your knowledge of Bigtable here, however I feel like I am missing something as to why this doesn't exist. It looks to me like a [valid RPC](https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.longrunning#google.longrunning.GetOperationRequest) that [mirrors Spanner](https://cloud.google.com/spanner/docs/reference/rpc/google.longrunning#google.longrunning.GetOperationRequest).

Reply:
It's not actually used anywhere.  Even if the RPC technically works, there are much better examples of methods that can be used to showcase the use the Instance.  Specifically, methods like `createInstance` and data operations, once we add them.

Reply:
I agree this isn't a great method to showcase the use of an instance. This documentation doesn't exist for that, however, it is used and add targeted samples for methods that are inherited from the [LROTrait](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Core/src/LongRunning/LROTrait.php).